src/libsync/discoveryphase.cpp

```diff
@@ -396,7 +396,11 @@ static void propertyMapToRemoteInfo(const QMap<QString, QString> &map, RemoteInf
         } else if (property == QLatin1String("getlastmodified")) {
             const auto date = QDateTime::fromString(value, Qt::RFC2822Date);
             Q_ASSERT(date.isValid());
-            result.modtime = date.toTime_t();
+            if (date.toSecsSinceEpoch() > 0) {
+                result.modtime = date.toSecsSinceEpoch();
+            } else {
+                result.modtime = 0;
```

Comment:
@mgallien You can get rid of `else` block if you move `result.modtime = 0;` before `if (date.toSecsSinceEpoch() > 0) {`, bu, just a nitpik

Reply:
yes good idea